base_attachment_object_storage/models/ir_attachment.py

```diff
@@ -65,6 +68,34 @@ def _register_hook(self):
         if update_module:
             self.env['ir.attachment'].sudo()._force_storage_to_object_storage()
 
+    @api.model
+    def _save_in_db_domain(self):
+        """Return a domain for attachments that must be forced to DB
+
+        Read the docstring of ``_save_in_db_anyway`` for more details.
+
+        The domain must be inline with the conditions in
+        ``_save_in_db_anyway``.
+        """
+        excluded_model_settings = self.env['ir.config_parameter'].sudo().\
+            get_param('excluded.models.storedb', default='')
+        excluded_model_for_db_store = excluded_model_settings.split(',')
+        mimetypes_settings = self.env['ir.config_parameter'].sudo().get_param(
+            'mimetypes.list.storedb', default='')
+        mimetypes_for_db_store = mimetypes_settings.split(',')
+        filesize = self.env['ir.config_parameter'].sudo().get_param(
+            'file.maxsize.storedb', default='0')
+
+        return [
+            '|',
+            # assets are stored in 'ir.ui.view'
+            ('res_model', '=', 'ir.ui.view'),
+            '&', '&',
+            ('file_size', '<', int(filesize)),
+            ('res_model', 'not in', excluded_model_for_db_store),
+            ('mimetype', '=like', mimetypes_for_db_store),
```

Comment:
Since you can have more than one mimetype, and we are looking for the beginning only...
`not in` can't work, `=like` on a list neither. Should we add one tuple per possible mimetype separated by `|`?

Would that work?
```
domain += ['|'] * len(mimetypes_for_db_store) - 1
domain += [('mimetype', '=like', mimetype) for mimetype in mimetypes_for_db_store]
```

Reply:
@guewen thanks for pointing it, I have done a fix, I have try both cases (no mimetypes + several ones), and it seems to works